src/main/java/edu/gatech/cs2340/trydent/internal/JavaFXManager.java

```diff
@@ -0,0 +1,31 @@
+package edu.gatech.cs2340.trydent.internal;
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+
+/**
+ * Class to manage starting/stopping JavaFX.
+ *
+ * @author Garrett Malmquist
+ */
+public interface JavaFXManager {
```

Comment:
Why is this public? Isn't this an interface for internal use only?
If we are making it public, then maybe we should re-factor it to a factory pattern for dependency injection later (not now).